xoxzo/cloudpy/cloudpy.py

```diff
@@ -0,0 +1,126 @@
+# xoxzo api libirary
+
+'''
+This is the API library for Xoxzo's telephony service.
+http://docs.xoxzo.com/en/
+
+see http://docs.python-requests.org/en/master/ for requests library.
+'''
+
+import requests
+import os
+
+__author__ = "Akira Nonaka <nonaka@mac.com>"
+__status__ = "development"
+__version__ = "0.1"
+__date__ = "30 March 2011"
+
+
+class XoxzoClient:
+    '''
+    Base class to access Xoxzo API.
+
+    :param string sid:  Your sid of the xoxzo account.
+        If None, value of the environment variable
+        XOXZO_API_SID will be used.
+    :param string auth_token: Your auth_token of the xoxzo account.
+        If None, value of the environment variable XOXZO_API_AUTH_TOKEN
+        will be used.
+    '''
```

Comment:
@anonaka Can you remove getting the credentials from env var ? It's an oversight of mine. Our library shouldn't really concern how user will get the credentials. It should use what being passed to `__init__()` as is. We should look into env var if we also provide a CLI command for the library.

I'm sorry for the back and forth.